B827EBFF8E245D.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gateway_conf": {
+      "gateway_ID": "B827EBFF8E245D",
+      "servers": [
+        {
+          "server_address": "router.us.thethings.network",
```

Comment:
this is the old v2 router address, please update to the correct v3 router/cluster